packages/gatsby-core-utils/src/index.ts

```diff
@@ -0,0 +1,8 @@
+export { default as createContentDigest } from "./create-content-digest"
+export { joinPath, isNodeInternalModulePath, slash } from "./path"
+export { default as cpuCoreCount } from "./cpu-core-count"
+export { resolve as urlResolve } from "./url"
+export { isCI, getCIName } from "./ci"
+export { default as createRequireFromPath } from "./create-require-from-path"
+export { default as getConfigStore } from "./get-config-store"
```

Comment:
This needs to be updated because [of this change](https://github.com/gatsbyjs/gatsby/pull/22051#discussion_r389332019)